frontend/src/metabase/components/LeftNavPane/LeftNavPane.jsx

```diff
@@ -39,9 +40,13 @@ export function LeftNavPaneItemBack({ path }) {
   );
 }
 
-export function LeftNavPane({ children }) {
+export function LeftNavPane({ children, fullHeight }) {
```

Comment:
```suggestion
export function LeftNavPane({ children, fullHeight = true }) {
```

Let's provide a default here to preserve the original behavior for other parent components that may be using `LeftNavPane` and to fix the type error we are seeing in `MetadataFieldSettings` below.